scripts/deployment/travis_before_deploy.sh

```diff
@@ -12,6 +12,32 @@ validate_env_var() {
     done
 }
 
+# ContentaCMS Files List Function
+# This function receives no argument
+# List of least minimum files needed for Contenta CMS to work
+contentacms_files_list() {
```

Comment:
I would rather have a black list than a white list. It will be very easy that we add something critical and forget that this code is deleting it.